users/tests.py

```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
+
```

Comment:
maybe add a simple test for your code so we'll know better if it works properly? 
or you're planning on doing it with the PR where you add the model part?